simul/platform/mininet.go

```diff
@@ -0,0 +1,392 @@
+// Mininet is the platform-implementation that uses the MiniNet-framework
+// set in place by Marc-Andre Luthi from EPFL. It is based on MiniNet,
+// as it uses a lot of similar routines
+
+package platform
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"net"
+
+	"io/ioutil"
+
+	"path"
+
+	"strings"
+
+	"sort"
+
+	"github.com/BurntSushi/toml"
+	"github.com/dedis/cothority/app/lib/config"
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/sda"
+)
+
+// MiniNet represents all the configuration that is necessary to run a simulation
+// on remote hosts running Mininet.
+type MiniNet struct {
+	// *** Mininet-related configuration
+	// The login on the platform
+	Login string
+	// The outside host on the platform
+	External string
+	// Directory we start - supposed to be `cothority/simul`
+	wd string
+	// Directory holding the cothority-go-file
+	cothorityDir string
+	// Directory storing the additional files
+	mininetDir string
+	// Directory for building
+	buildDir string
+	// IPs of all hosts
+	HostIPs []string
+	// Channel to communicate stopping of experiment
+	sshMininet chan string
+	// Whether the simulation is started
+	started bool
+	// RC-configuration
+	config string
+
+	// ProxyAddress : the proxy will redirect every traffic it
+	// receives to this address
+	ProxyAddress string
+	// Port number of the monitor and the proxy
+	MonitorPort int
+
+	// Simulation to be run
+	Simulation string
+	// Number of servers to be used
+	Servers int
+	// Number of machines
+	Hosts int
+	// Debugging-level: 0 is none - 5 is everything
+	Debug int
+	// Whether to show time in debugging messages
+	DebugTime bool
+	// Whether to show color debugging-messages
+	DebugColor bool
+	// The number of seconds to wait for closing the connection
+	RunWait int
+	// Delay in ms of the network connection
+	Delay int
+	// Bandwidth in Mbps of the network connection
+	Bandwidth int
+}
```

Comment:
This struct is almost identical to the deterlab one: https://github.com/dedis/cothority/blob/master/simul/platform/deterlab.go#L38

I wonder if we should refactor out a common struct instead of we are currently doing: basically copy&paste plus modifying comments and adding/removing very few fields. 
This might be out of scope of this PR... Do you think it's worth it to open an issue for that (or add it to an already existing more general refactoring issue)?